src/create-config.ts

```diff
@@ -0,0 +1,317 @@
+        .div(10000);
+      const beaconUpdateSubscriptionConditionParameters = ethers.utils.defaultAbiCoder.encode(['uint256'], [threshold]);
+      const encodedBeaconUpdateSubscriptionConditions = encode([
+        {
+          type: 'bytes32',
+          name: '_conditionFunctionId',
+          value: ethers.utils.defaultAbiCoder.encode(
+            ['bytes4'],
+            [DapiServerInteface.getSighash('conditionPspBeaconUpdate')]
+          ),
+        },
+        { type: 'bytes', name: '_conditionParameters', value: beaconUpdateSubscriptionConditionParameters },
+      ]);
+
+      return beacon.chains.map((chain) => {
+        const chainId = chainNameToChainId[chain.name] || 1;
+        //TODO: Add DapiServer contracts based on chain
+        const DapiServerAddress = ethers.constants.AddressZero;
+        const sponsor = chain.sponsor;
+        const beaconUpdateSubscriptionId = ethers.utils.keccak256(
+          ethers.utils.defaultAbiCoder.encode(
+            ['uint256', 'address', 'bytes32', 'bytes', 'bytes', 'address', 'address', 'address', 'bytes4'],
+            [
+              chainId,
+              airnodeAddress,
+              templateId,
+              parameters,
+              encodedBeaconUpdateSubscriptionConditions,
+              airnodeAddress,
+              sponsor,
+              DapiServerAddress,
+              DapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+            ]
+          )
+        );
+        return {
+          [beaconUpdateSubscriptionId]: {
+            chainId: `${chainId}`,
+            parameters,
+            airnodeAddress,
+            templateId,
+            conditions: encodedBeaconUpdateSubscriptionConditions,
+            relayer: airnodeAddress,
+            sponsor,
+            requester: DapiServerAddress,
+            fulfillFunctionId: DapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+          },
+        };
+      });
+    })
+    .reduce((subscriptionsObject, subscription) => ({ ...subscriptionsObject, ...subscription }), {});
+
+  const airkeeperTriggers = {
+    rrpBeaconServerKeeperJobs: [],
+    protoPsp: Object.keys(airkeeperSubscriptions),
+  };
+
+  const airkeeperTemplates = Object.values(apiData.templates).reduce(
+    (templateObj, template) => ({
+      ...templateObj,
+      [template.templateId]: {
+        endpointId: template.endpointId,
+        templateParameters: template.parameters,
+      },
+    }),
+    {}
+  );
+
+  const airkeeperEndpointArray = await Promise.all(
+    Object.values(apiData.ois).flatMap((ois) => {
+      return ois.endpoints.map(async (endpoint) => ({
+        [await deriveEndpointId(ois.title, endpoint.name)]: { endpointName: endpoint.name, oisTitle: ois.title },
+      }));
+    })
+  );
+
+  const AirkeeperEndpoints = airkeeperEndpointArray.reduce(
+    (endpointsObject, endpoint) => ({ ...endpointsObject, ...endpoint }),
+    {}
+  );
+
+  const AirkeeperConfig: AirkeeperConfig = {
```

Comment:
As per the other comment further up, this type can also be removed because this object is used in the `OperationsRepository` type (you assign this object when you build the ops repo compound object).